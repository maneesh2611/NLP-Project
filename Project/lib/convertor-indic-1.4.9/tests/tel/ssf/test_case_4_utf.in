<Sentence id="1">			
1	((	NP	
1.1	సూపర్	NN	<fs af='సూపర్,n,,sg,,d,0,0'>|<fs af='సూపర్,n,,sg,,o,టి,ti'>
	))		
2	((	NP	
2.1	స్టార్	NN	<fs af='స్టార్,n,,sg,,d,0,0'>|<fs af='స్టార్,n,,sg,,o,టి,ti'>
	))		
3	((	NP	
3.1	ననే	PRP	<fs af='నన,n,,sg,,,0,0_e' poslcat="NM">
	))		
4	((	NP	
4.1	అహం	NN	<fs af='అహం,n,,sg,,d,0,0'>
	))		
5	((	VGNF	
5.1	లేకుండా	VM	<fs af='లేకుండు,v,any,any,any,,ఇ_ఆ,i_A'>
	))		
6	((	NP	
6.1	ఆయన	PRP	<fs af='ఆయన,pn,,sg,,d,0,0'>|<fs af='ఆయన,pn,,sg,,o,టి,ti'>
	))		
7	((	VGNF	
7.1	చూపిన	VM	<fs af='చూపు,v,any,any,any,,ఇన,ina'>
	))		
8	((	NP	
8.1	ఆప్యాయతకు	NN	<fs af='ఆప్యాయత,n,,sg,,,కి,ki'>
	))		
9	((	VGNF	
9.1	కదిలిన	VM	<fs af='కదులు,v,any,any,any,,ఇన,ina'>
	))		
10	((	VGNF	
10.1	పోయిన	VM	<fs af='పో,v,any,any,any,,ఇన,ina'>
	))		
11	((	NP	
11.1	యూనిట్	NN	<fs af='యూనిట్,n,,sg,,d,0,0'>|<fs af='యూనిట్,n,,sg,,o,టి,ti'>
	))		
12	((	NP	
12.1	సభ్యులు	NN	<fs af='సభ్యుడు,n,,pl,,d,0,0'>|<fs af='సభ్యుడు,n,,pl,,d,వు,vu'>
	))		
13	((	NP	
13.1	రజనీ	NNP	<fs af='రజని,n,,sg,,,0,0_V'>
13.2	,	SYM	<fs af=',,punc,,,,,' poslcat="NM">
	))		
14	((	NP	
14.1	ఆయన	PRP	<fs af='ఆయన,pn,,sg,,d,0,0'>|<fs af='ఆయన,pn,,sg,,o,టి,ti'>
	))		
15	((	NP	
15.1	కుటుంబ	NN	<fs af='కుటుంబం,n,,sg,,o,టి,ti'>
	))		
16	((	NP	
16.1	సభ్యులు	NN	<fs af='సభ్యుడు,n,,pl,,d,0,0'>|<fs af='సభ్యుడు,n,,pl,,d,వు,vu'>
	))		
17	((	NP	
17.1	పదికాలాలు	NN	<fs af='పదికాలాలు,unk,,,,,,' poslcat="NM">
	))		
18	((	NP	
18.1	పాటు	NN	<fs af='పాటు,n,,sg,,d,0,0'>|<fs af='పాటు,n,,sg,,o,టి,ti'>
	))		
19	((	RBP	
19.1	చల్లగా	RB	<fs af='చల్లగా,avy,,,,,0,0_avy' poslcat="NM">|<fs af='చల్లగా,avy,,,,,0,0_avy' poslcat="NM">|<fs af='చల్ల,n,,sg,,,గా,gA' poslcat="NM">|<fs af='చల్ల,adj,,sg,,,గా,gA_adj' poslcat="NM">|<fs af='చల్లు,v,any,any,any,,అ,a_gA' poslcat="NM">
	))		
20	((	VGNF	
20.1	ఉండాలని	VM	<fs af='ఉండు,v,any,any,any,,ఆలి_అని,Ali_ani'>
	))		
21	((	VGNF	
21.1	కోరుకుంటూ	VM	<fs af='కోరు,v,any,any,any,,కొన్_తూ,koVn_wU'>
	))		
22	((	NP	
22.1	తమ	PRP	<fs af='తాము,pn,any,sg,2,o,టి,ti'>
	))		
23	((	NP	
23.1	కృతజ్ఞతలు	NN	<fs af='కృతజ్ఞత,n,,pl,,d,0,0'>|<fs af='కృతజ్ఞత,n,,pl,,d,వు,vu'>
	))		
24	((	VGF	
24.1	చాటుకున్నారు	VM	<fs af='చాటు,v,fm,pl,3,,కొన్_ఆ,koVn_A'>
24.2	।	SYM	<fs af='।,punc,,,,,,'>
	))		
</Sentence>			
