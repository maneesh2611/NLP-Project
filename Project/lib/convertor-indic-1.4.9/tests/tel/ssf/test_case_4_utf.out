<Sentence id="1">			
1	((	NP	
1.1	sUpar	NN	<fs af='sUpar,n,,sg,,d,0,0'>|<fs af='sUpar,n,,sg,,o,ti,ti'>
	))		
2	((	NP	
2.1	stAr	NN	<fs af='stAr,n,,sg,,d,0,0'>|<fs af='stAr,n,,sg,,o,ti,ti'>
	))		
3	((	NP	
3.1	nane	PRP	<fs af='nana,n,,sg,,,0,0_e' poslcat="NM">
	))		
4	((	NP	
4.1	ahaM	NN	<fs af='ahaM,n,,sg,,d,0,0'>
	))		
5	((	VGNF	
5.1	lekuMdA	VM	<fs af='lekuMdu,v,any,any,any,,i_A,i_A'>
	))		
6	((	NP	
6.1	Ayana	PRP	<fs af='Ayana,pn,,sg,,d,0,0'>|<fs af='Ayana,pn,,sg,,o,ti,ti'>
	))		
7	((	VGNF	
7.1	cUpina	VM	<fs af='cUpu,v,any,any,any,,ina,ina'>
	))		
8	((	NP	
8.1	ApyAyawaku	NN	<fs af='ApyAyawa,n,,sg,,,ki,ki'>
	))		
9	((	VGNF	
9.1	kaxilina	VM	<fs af='kaxulu,v,any,any,any,,ina,ina'>
	))		
10	((	VGNF	
10.1	poyina	VM	<fs af='po,v,any,any,any,,ina,ina'>
	))		
11	((	NP	
11.1	yUnit	NN	<fs af='yUnit,n,,sg,,d,0,0'>|<fs af='yUnit,n,,sg,,o,ti,ti'>
	))		
12	((	NP	
12.1	saByulu	NN	<fs af='saByudu,n,,pl,,d,0,0'>|<fs af='saByudu,n,,pl,,d,vu,vu'>
	))		
13	((	NP	
13.1	rajanI	NNP	<fs af='rajani,n,,sg,,,0,0_V'>
13.2	,	SYM	<fs af=',,punc,,,,,' poslcat="NM">
	))		
14	((	NP	
14.1	Ayana	PRP	<fs af='Ayana,pn,,sg,,d,0,0'>|<fs af='Ayana,pn,,sg,,o,ti,ti'>
	))		
15	((	NP	
15.1	kutuMba	NN	<fs af='kutuMbaM,n,,sg,,o,ti,ti'>
	))		
16	((	NP	
16.1	saByulu	NN	<fs af='saByudu,n,,pl,,d,0,0'>|<fs af='saByudu,n,,pl,,d,vu,vu'>
	))		
17	((	NP	
17.1	paxikAlAlu	NN	<fs af='paxikAlAlu,unk,,,,,,' poslcat="NM">
	))		
18	((	NP	
18.1	pAtu	NN	<fs af='pAtu,n,,sg,,d,0,0'>|<fs af='pAtu,n,,sg,,o,ti,ti'>
	))		
19	((	RBP	
19.1	callagA	RB	<fs af='callagA,avy,,,,,0,0_avy' poslcat="NM">|<fs af='callagA,avy,,,,,0,0_avy' poslcat="NM">|<fs af='calla,n,,sg,,,gA,gA' poslcat="NM">|<fs af='calla,adj,,sg,,,gA,gA_adj' poslcat="NM">|<fs af='callu,v,any,any,any,,a,a_gA' poslcat="NM">
	))		
20	((	VGNF	
20.1	uMdAlani	VM	<fs af='uMdu,v,any,any,any,,Ali_ani,Ali_ani'>
	))		
21	((	VGNF	
21.1	korukuMtU	VM	<fs af='koru,v,any,any,any,,koVn_wU,koVn_wU'>
	))		
22	((	NP	
22.1	wama	PRP	<fs af='wAmu,pn,any,sg,2,o,ti,ti'>
	))		
23	((	NP	
23.1	kqwajFawalu	NN	<fs af='kqwajFawa,n,,pl,,d,0,0'>|<fs af='kqwajFawa,n,,pl,,d,vu,vu'>
	))		
24	((	VGF	
24.1	cAtukunnAru	VM	<fs af='cAtu,v,fm,pl,3,,koVn_A,koVn_A'>
24.2	ред	SYM	<fs af='ред,punc,,,,,,'>
	))		
</Sentence>			
